port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -455,6 +456,10 @@ async def sync_raw_all(
                         event.on_abort(lambda: task.cancel())
 
                         creation_results.append(await task)
+                try:
+                    await event.failed_entity_handler.handle_failed()
+                except:
+                    await event.failed_entity_handler.handle_failed_no_sort()
```

Comment:
maybe something like this
```suggestion
                try:
                    for entity in event.failed_entity_handler.get_sorted()
                         await self.upsert_entity(
                    entity,
                    request_options,
                    user_agent_type,
                    should_raise=should_raise,
                )
                for entity in entities
            ),
                except:
                    await event.failed_entity_handler.handle_failed_no_sort()
```